controller_interface/test/test_controller_with_options.hpp

```diff
@@ -44,7 +44,7 @@ class ControllerWithOptions : public controller_interface::ControllerInterface
   {
     rclcpp::NodeOptions options;
     options.allow_undeclared_parameters(true).automatically_declare_parameters_from_overrides(true);
-    node_ = std::make_shared<rclcpp_lifecycle::LifecycleNode>(controller_name, namespace_, options);
+    set_node(std::make_shared<rclcpp_lifecycle::LifecycleNode>(controller_name, options));
```

Comment:
We would still need to pass namespace to the node creation.

Reply:
Not sure why I removed that, adding it back in.